src/status_im2/common/scroll_page/view.cljs

```diff
@@ -1,37 +1,24 @@
 (ns status-im2.common.scroll-page.view
   (:require [oops.core :as oops]
             [quo2.core :as quo]
+            [quo.react]
             [quo2.foundations.colors :as colors]
             [react-native.core :as rn]
             [react-native.platform :as platform]
             [reagent.core :as reagent]
             [status-im2.common.scroll-page.style :as style]
-            [utils.re-frame :as rf]))
+            [utils.re-frame :as rf]
+            [react-native.reanimated :as reanimated]
+            [quo.animated :as animated]))
```

Comment:
I thought `quo.animated` shouldn't be used in the new `status-im2`, but I'm not sure tbh. Shouldn't we just use `react-native.reanimated`?

Reply:
yes, we should use reanimated.